### ISSUE TITLE
How to reach the write apis, only read apis here.

### ISSUE THREAD

**Issue by Alpha2J** [starkware-libs/starknet-specs#101]
![image](https://github.com/starkware-libs/starknet-specs/assets/17811469/0839705a-05fa-406b-8c64-a6e7eb8e6aea)

**Comment by Mirko-von-Leipzig** [comment]
The write APIs aren't hosted as part of the playground, so you would have to just read the [raw spec](https://github.com/starkware-libs/starknet-specs/blob/master/api/starknet_write_api.json).

**Comment by Alpha2J** [comment]
But it contains numerous references, making it challenging to navigate. Are there any tools available to parse them?
![image](https://github.com/starkware-libs/starknet-specs/assets/17811469/d95fa2ea-cb07-40f3-b798-a6e8ddc1f2af)

**Comment by Alpha2J** [comment]
I have tried [open-rpc playground](https://github.com/open-rpc/playground) locally, but it did not work.

**Comment by Mirko-von-Leipzig** [comment]
Yeah the issue is that the write api also references components/types defined in the read api file. However the playground doesn't know how to link files like that.
